src/test/ui/deprecation/deprecation-in-future.rs

```diff
@@ -0,0 +1,16 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![deny(deprecated_in_future)]
+
+#[deprecated(since = "99.99.99", note = "text")]
+pub fn deprecated_future() {}
+
+fn test() {
+    deprecated_future(); //~ ERROR use of item that will be deprecated in a future version 99.99.99 'deprecated_future': text
```

Comment:
The item name seems oddly out of place where it is.

What about
```
use of item 'deprecated_future' that will be deprecated in a future version 99.99.99: text
```

Reply:
Adjusted